src/main/java/com/puppycrawl/tools/checkstyle/site/AllCheckSummaries.java

```diff
@@ -48,6 +48,16 @@
  * Macro to generate table rows for all Checkstyle modules.
  * Includes every Check.java file that has a Javadoc.
  * Uses href path structure based on src/site/xdoc/checks.
+ *
+ * <p>Supports optional "package" parameter to filter checks by package.
+ * When package parameter is provided, only checks from that package are included.
+ * Usage:
+ * <pre>
+ * &lt;macro name="allCheckSummaries"/&gt;  (all checks)
+ * &lt;macro name="allCheckSummaries"&gt;
+ *   &lt;param name="package" value="annotation"/&gt;
+ * &lt;/macro&gt;
+ * </pre>
```

Comment:
Minor suggestion to improve readability and separation:

```suggestion
 * Macro to generate table rows for all Checkstyle modules.
 * Includes every Check.java file that has a Javadoc.
 * Uses href path structure based on src/site/xdoc/checks.
 * Usage:
 * <pre>
 * &lt;macro name="allCheckSummaries"/&gt;
 * </pre>
 *
 * <p>Supports optional "package" parameter to filter checks by package.
 * When package parameter is provided, only checks from that package are included.
 * Usage:
 * <pre>
 * &lt;macro name="allCheckSummaries"&gt;
 *   &lt;param name="package" value="annotation"/&gt;
 * &lt;/macro&gt;
 * </pre>
```

Reply:
Done

Reply:
Resolved.